src/core/Breadcrumbs/Breadcrumbs.test.tsx

```diff
@@ -98,29 +98,18 @@ it('should overflow when there is not enough space', () => {
   expect(breadcrumbs[1].textContent).toEqual('…');
   expect(breadcrumbs[1].firstElementChild?.classList).toContain('iui-ellipsis');
   expect(breadcrumbs[2].textContent).toEqual('Item 2');
-
-  scrollWidthSpy.mockRestore();
-  offsetWidthSpy.mockRestore();
 });
 
 it('should restore hidden items when there is enough space again', () => {
```

Comment:
Not sure this test is needed as it was testing the inside logic of `useOverflow` hook but I leaved it.

Reply:
Removed.